doc/source/install.rst

```diff
@@ -7,13 +7,25 @@ This section covers the basics of how to download and install TomoPy.
 .. contents:: Contents:
    :local:
 
+
 Supported Environments
-======================
+----------------------
 
 TomoPy is tested, built, and distributed for python 2.7 3.5 3.6 on Linux/macOS
 and python 3.5 3.6 on Windows 10.
 
-Installing from Conda (Recommended)
+Common issues
+-------------
+
+No issues with the current build system have been reported.
+
+Importing TomoPy
+----------------
+
+When importing, it is best to import TomoPy before importing numpy.
+See `this thread <https://github.com/tomopy/tomopy/issues/178>`_ for details.
+
```

Comment:
Because we stopped FFTW as a dependency, Importing TomoPy section can be removed. I'd put this part under Common issues as a remark for users, who use FFTW instead of MKL-FFT.